.cookiecutter.json

```diff
@@ -0,0 +1,25 @@
+{
+  "cookiecutter": {
+    "codeowner_github_usernames": "@glennmatthews @jathanism @lampwins @chadell",
```

Comment:
Should this be `@nautobot/plugin-ssot` instead?

Reply:
Again, this was generated when the cookie was first created and I'm not sure it needs to be updated as we update the project?